### ISSUE TITLE
On Reflow "Awesome-Azd" page is accessible with 2 scroll bars: A11y_Awesome AZD_ Template library_Search_Reflow.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#343]
### Tags:
#A11yTCS; #A11ySev3; #A11yMAS; #WCP; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #Feature_AwesomeAZD_Web_Mar2024; #WCAG1.4.10; #Reflow; #DesktopWeb; #A11yWCAG2.1; #Zoom;

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20 
Windows 11 Enterprise.
Version 23H2 OS build 22631.2050.
### Repro Steps: 
1. Hit this URL: https://azure.github.io/awesome-azd/.
2. Now apply reflow.
3. Now verify the issue that on reflow page is accessible with 2 scroll bars or not.
### Actual Result:
On Reflow "Awesome-Azd" page is accessible with 2 scroll bars.
### Expected Results:
On Reflow the "Awesome-Azd" page should be accessible with either horizontal or vertical scroll bar without any truncations and overlapping.
### User Impact:
Low visibility users will face difficulty to access the page data if it is accessible with 2 scroll bars.

### Attachment:

https://github.com/Azure/awesome-azd/assets/100270074/ed94f431-7db7-453d-9748-bb99ddfd1f5e

**Comment by hemarina** [comment]
Closing this issue as we made a whole update on the site, and this issue is not update to date with existing site. @Harika1702 Feel free to reopen this issue if needed.
